app/scripts/controllers/newOverview.js

```diff
@@ -0,0 +1,1200 @@
+
+    watches.push(DataService.watch({
+      group: "extensions",
+      resource: "replicasets"
+    }, context, function(replicaSetData) {
+      overview.replicaSets = replicaSetData.by('metadata.name');
+      groupPods();
+      groupReplicaSets();
+      updateServicesForObjects(overview.vanillaReplicaSets);
+      updateServicesForObjects(overview.monopods);
+      updatePodWarnings(overview.vanillaReplicaSets);
+      updateLabelSuggestions(overview.vanillaReplicaSets);
+      updateFilter();
+      Logger.log("replicasets (subscribe)", overview.replicaSets);
+    }));
+
+    watches.push(DataService.watch({
+      group: "apps",
+      resource: "statefulsets"
+    }, context, function(statefulSetData) {
+      overview.statefulSets = statefulSetData.by('metadata.name');
+      groupPods();
+      updateServicesForObjects(overview.statefulSets);
+      updateServicesForObjects(overview.monopods);
+      updatePodWarnings(overview.statefulSets);
+      updateLabelSuggestions(overview.statefulSets);
+      updateFilter();
+      Logger.log("statefulsets (subscribe)", overview.statefulSets);
+    }, {poll: limitWatches, pollInterval: DEFAULT_POLL_INTERVAL}));
+
+    watches.push(DataService.watch({
+      group: "extensions",
+      resource: "deployments"
+    }, context, function(deploymentData) {
+      overview.deployments = deploymentData.by('metadata.name');
+      groupReplicaSets();
+      updateServicesForObjects(overview.deployments);
+      updateServicesForObjects(overview.vanillaReplicaSets);
+      updateLabelSuggestions(overview.deployments);
+      updateFilter();
+      Logger.log("deployments (subscribe)", overview.deployments);
+    }));
+
+    watches.push(DataService.watch({
+      group: "extensions",
+      resource: "horizontalpodautoscalers"
+    }, context, function(hpaData) {
+      overview.horizontalPodAutoscalers = hpaData.by("metadata.name");
+      groupHPAs();
+      Logger.log("autoscalers (subscribe)", overview.horizontalPodAutoscalers);
+    }, {poll: limitWatches, pollInterval: 60 * 1000}));
+
+    // Always poll quotas instead of watching, its not worth the overhead of maintaining websocket connections
+    watches.push(DataService.watch('resourcequotas', context, function(quotaData) {
+      state.quotas = quotaData.by("metadata.name");
+      updateQuotaWarnings();
+    }, {poll: true, pollInterval: 60 * 1000}));
+
+    watches.push(DataService.watch('appliedclusterresourcequotas', context, function(clusterQuotaData) {
+      state.clusterQuotas = clusterQuotaData.by("metadata.name");
+      updateQuotaWarnings();
+    }, {poll: true, pollInterval: 60 * 1000}));
+
+    // List limit ranges in this project to determine if there is a default
+    // CPU request for autoscaling.
+    DataService.list("limitranges", context, function(response) {
+      state.limitRanges = response.by("metadata.name");
+    });
+
+    watches.push(DataService.watch("imagestreams", context, function(imageStreamData) {
```

Comment:
thinking we should have all the watches first, then the limited watches, then the always polled, then the lists, and then the gets.  I think most of them were in that order besides this one.